packages/xrpc/src/util.ts

```diff
@@ -44,7 +44,9 @@ export function constructMethodCallUri(
       if (!paramSchema) {
         throw new Error(`Invalid query parameter: ${key}`)
       }
-      uri.searchParams.set(key, encodeQueryParam(paramSchema.type, value))
+      if (value !== undefined) {
```

Comment:
Without this change, if you pass `undefined` as a parameter it would make it into the query as `?param=undefined`.

Reply:
nice 👌